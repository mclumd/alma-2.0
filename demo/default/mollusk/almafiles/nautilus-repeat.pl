rel(nautilus, steve).
if(now(2), rel(nautilus, sonny)).
if(now(4), rel(nautilus, sam)).
if(now(6), rel(nautilus, sue)).
