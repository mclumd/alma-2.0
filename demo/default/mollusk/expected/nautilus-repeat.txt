rel(shell_bearer, steve)
abnormal(steve, cephalopod, "rel(shell_bearer, steve)")
rel(mollusk, steve)
rel(cephalopod, steve)
rel(nautilus, steve)
rel(shell_bearer, sonny)
abnormal(sonny, cephalopod, "rel(shell_bearer, sonny)")
rel(mollusk, sonny)
rel(cephalopod, sonny)
rel(nautilus, sonny)
rel(shell_bearer, sam)
abnormal(sam, cephalopod, "rel(shell_bearer, sam)")
rel(mollusk, sam)
rel(cephalopod, sam)
rel(nautilus, sam)
rel(shell_bearer, sue)
abnormal(sue, cephalopod, "rel(shell_bearer, sue)")
rel(mollusk, sue)
rel(cephalopod, sue)
rel(nautilus, sue)
